packages/libs/wdk-client/src/StoreModules/RecordStoreModule.ts

```diff
@@ -84,11 +84,22 @@ export function reduce(state: State = {} as State, action: Action): State {
     case RECORD_RECEIVED: {
       if (action.id !== state.requestId) return state;
       let { record, recordClass, categoryTree } = action.payload;
+
+      const collapsedSections = getLeaves(categoryTree, (node) => node.children)
+        .filter(
+          (
+            node
+          ): node is CategoryTreeNode & { properties: { name: string[] } } =>
+            !!node.properties.scope?.includes('record-collapsed') &&
+            !!node.properties.name
+        )
+        .map((node) => node.properties.name[0]);
```

Comment:
Is there ever >1 element in the `name` array?

Reply:
I don't think so, but the ontology properties are always multi-value.

FYI, you can use this function:
https://github.com/VEuPathDB/web-monorepo/blob/c02d237d2116a452a8195a51ad570b10e6ccdb2e/packages/libs/wdk-client/src/Utils/CategoryUtils.tsx#L91-L93